_pages/integrations/queue-manager.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: article
+title: "Queue Manager"
+excerpt-short: "Set up broker connections and topics that can be enabled to process algorithm data"
+categories: [integrations]
+tags: [integrations]
+show_related: true
+---
+
+These directions will help admins set up broker connections and topics, and help users enable queues to process their algorithms.
+
+This is only available for Enterprise installations of Algorithmia.
+{: .notice-enterprise}
+
+## Creating and Configuring a Broker Connection in the Algorithmia platform
+
+Admins can set up connections to configured queue managers on external clusters that the customer owns.
+
+Open the Algorithmia Web user interface and log in with an admin account.
+
+Under the Admin menu, click on the "Queue manager" menu item (under the "System actions" category) to see a list of configured broker connections.
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/queue-manager/broker-connection-list.png">
+
+To create a new broker connection, click on the "Connect queue" button. This will bring up a form where the admin will fill in the proper configuration options. When filled in, click on the "Connect to Broker" button to make the connection. When successful, the admin will see the new connection in the list of configured brokers. 
+
+Currently, the only broker type supported is Kafka.
+{: .notice-info}
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/queue-manager/broker-connection-create.png">
+
+Click on a configured broker connection in the list to view it's details and list of topics set up for this broker connection. This details page is where an admin will also configure the algorithms that have access to these topics. Also on this page is an action button to delete the broker connection. Clicking it will present a confirmation dialog. After confirming, the broker connection will be removed from the list and unavailable for algorithms to use.
```

Comment:
```suggestion
Click on a configured broker connection in the list to view its details and list of topics set up for this broker connection. This details page is where an admin will also configure the algorithms that have access to these topics. Also on this page is an action button to delete the broker connection. Clicking it will present a confirmation dialog. After confirming, the broker connection will be removed from the list and unavailable for algorithms to use.
```